.github/workflows/release-homebrew.yaml

```diff
@@ -50,8 +50,11 @@ jobs:
     timeout-minutes: 60
     env:
       FINCH_TAG: ${{ needs.get-latest-tag.outputs.tag }}
-      FINCH_VESRION: ${{ needs.get-latest-tag.outputs.version }}
+      FINCH_VERSION: ${{ needs.get-latest-tag.outputs.version }}
     steps:
+      - uses: actions/setup-go@v3
```

Comment:
Do we need to setup `go` for `get-latest-tag` job?

Reply:
No I don't think so. Here is an example of using the action `WyriHaximus/github-action-get-previous-tag` https://github.com/WyriHaximus/github-action-get-previous-tag#example

Reply:
Okay, then maybe we can just remove the `- uses: actions/setup-go@v3` step.

Reply:
The setup-go action is used in the `macos-arm64-test-installer` and `macos-amd64-test-installer` jobs, which is same as the [`release-installer` workflow](https://github.com/runfinch/finch/blob/main/.github/workflows/release-installer.yaml#L42). It should work without the `setup-go` action because self-hosted runners have installed `go` via brew. But it might be used to set up go env when `go` is not installed.

Reply:
My bad, I thought `uses: actions/setup-go@v3` was in `get-latest-tag` job :(. Yeah you are right, its needed for `macos-arm64-test-installer` and `macos-amd64-test-installer`.